models/RoutesParser.cfc

```diff
@@ -592,6 +612,27 @@ component accessors="true" threadsafe singleton {
 		}
 	}
 
+	/**
+	* return true if the metadata represents a function marked "noCbSwagger"
+	*/
+	private boolean function isAnnotatedWithTruthyNoCbSwaggerAttribute( required struct functionMetadata ) {
+		if ( !structKeyExists( functionMetadata, "noCbSwagger" ) ) {
+			return false;
+		}
+
+		if ( trim( functionMetadata.noCbSwagger ) == "" ) {
```

Comment:
You can simplify the conditionals here with a ternary here:

```
return  !structKeyExists( functionMetadata, "cbSwagger" )
             ? false
             :  !functionMetadata.cbswagger.len() 
                    ||  ( 
                            isValid( "boolean", functionMetadata.noCbSwagger ) && functionMetadata.noCbSwagger 
                         );

```

Reply:
addressed